integration/localnet/bootstrap.go

```diff
@@ -503,37 +474,12 @@ func prepareObserverService(i int, observerName string, agPublicKey string) Serv
 		fmt.Sprintf("--rpc-addr=%s:%d", observerName, RPCPort),
 		fmt.Sprintf("--secure-rpc-addr=%s:%d", observerName, SecuredRPCPort),
 		fmt.Sprintf("--http-addr=%s:%d", observerName, HTTPPort),
-		"--bootstrapdir=/bootstrap",
-		"--datadir=/data/protocol",
-		"--secretsdir=/data/secret",
-		"--loglevel=DEBUG",
-		fmt.Sprintf("--profiler-enabled=%t", profiler),
-		fmt.Sprintf("--tracer-enabled=%t", tracing),
-		"--profiler-dir=/profiler",
-		"--profiler-interval=2m",
-	}
-	observerService.Volumes = []string{
-		fmt.Sprintf("%s:/bootstrap:z", BootstrapDir),
+	}...)
+	observerService.Volumes = append(observerService.Volumes, []string{
 		fmt.Sprintf("%s:/profiler:z", profilerDir),
```

Comment:
i feel dumb.  why can't these be dedup?  just pass in extra arguments to the defaultService function ....

Reply:
it's not you being dumb -- it's me being lazy =). Thanks for raising the bar!